onsstove/onsstove.py

```diff
@@ -579,4 +579,8 @@ def lives_saved(self):
 
         total_pop = self.gdf['Calibrated_pop'].sum()
 
-        self.gdf["deaths_avoided"] = self.gdf.apply(lambda row: self.techs[row['final_tech']].deaths_avoided * row['Calibrated_pop'] / total_pop)
\ No newline at end of file
+        self.gdf["deaths_avoided"] = self.gdf.apply(lambda row: self.techs[row['final_tech']].deaths_avoided * row['Calibrated_pop'] / total_pop)
+
+    def reduced_emissions(self):
+
+        self.gdf["emissions_saved"] = self.gdf.apply(lambda row: self.techs[row['final_tech']].decreased_carbon_emissions)
```

Comment:
I think `decreased_carbon_emissions` is a series, so we need to pull only the value of the row, like:
```python
self.techs[row['final_tech']].decreased_carbon_emissions[row.index]
```